### PR TITLE
add concourse pipeline, fix test that randomly fail

### PR DESCRIPTION
**Pull request by MalpenZibo**

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - get: master-branch
+        trigger: true
+      - task: sbt-test
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: buildo/scala-sbt-alpine
+              tag: 8u201_2.12.11_1.3.9
```
Comment (mcastellini) [inline ci/pipeline.yml:23]: The version of scala included here does not match with the one declared in build.sbt. Also, I'm wondering why there isn't a `project/build.properties` file in which is defined `sbt.version`

File: ci/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+resources:
+  - name: master-branch
+    type: git
+    source:
+      branch: master
+      uri: git@github.com:MalpenZibo/rps.git
+      private_key: ((repo-key))
+      depth: 1
```
Comment (mcastellini) [inline ci/pipeline.yml:9]: I'm not really sure that `depth` should be declared here

File: ci/pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          image_resource:
+            type: docker-image
+            source:
+              repository: buildo/scala-sbt-alpine
+              tag: 8u201_2.12.11_1.3.9
+          inputs:
+            - name: master-branch
+          caches:
+            - path: .sbt
+            - path: .ivy2
```
Comment (mcastellini) [inline ci/pipeline.yml:28]: I'm not 100% sure that this cached path is correctly picked up by SBT

File: ci/pipeline.yml (hunk 1 above)
Comment (MalpenZibo) [reply ci/pipeline.yml:23]: It is not on the main repo so I haven't tracked on git. I will add that file in the next commit

File: ci/pipeline.yml (hunk 3 above)
Comment (MalpenZibo) [reply ci/pipeline.yml:28]: I had the same doubt. But the documentation is not very clear so I haven't found a good example

File: ci/pipeline.yml (hunk 2 above)
Comment (MalpenZibo) [reply ci/pipeline.yml:9]: wops! maybe params: {depth: 100}?

Comment (mcastellini) [reply ci/pipeline.yml:9]: Yep, but on the `get`.

**Pull request merged**